pixl_dcmd/src/pixl_dcmd/_dicom_helpers.py

```diff
@@ -54,10 +54,11 @@ def __init__(self, edition: str = "current"):
         self.dicom_info = EditionReader.load_dicom_info(json_path)
 
     def validate_original(self, dataset: Dataset) -> None:
-        # Temporarily disable logging to avoid spamming the console
-        logging.disable(logging.ERROR)
-        self.original_errors = IODValidator(dataset, self.dicom_info).validate()
-        logging.disable(logging.NOTSET)
+        self.original_errors = IODValidator(
+            dataset,
+            self.dicom_info,
+            log_level=logging.ERROR,
+        ).validate()
```

Comment:
not sure why the `IODValidator` logs were appearing in our docker logs after #512 but this seems to prevent that

Reply:
Yeah that's more reasonable way to fix it